pkg/app/piped/driftdetector/kubernetes/detector.go

```diff
@@ -193,9 +193,17 @@ func (d *detector) checkApplication(ctx context.Context, app *model.Application,
 		return err
 	}
 
-	ignoreFields := make([]string, 0)
+	ignoreConfig := make(map[string][]string, 0)
 	if ddCfg != nil {
-		ignoreFields = ddCfg.IgnoreFields
+		for _, ignoreField := range ddCfg.IgnoreFields {
+			// ignoreField is 'apiVersion:kind:namespace:name#fieldPath'
```

Comment:
Should add this line to the config/application as well to mention what we expect as driftDetection.ignoreFields

Reply:
I changed it. How about that?

Reply:
👍